requirements/requirements_docs.txt

```
@@ -1,7 +1,7 @@
 docutils>=0.17
 sphinx==5.2.3
 sphinxcontrib-trio~=1.1.2
-sphinx-hoverxref==1.2.0
+sphinx-hoverxref @ git+https://github.com/readthedocs/sphinx-hoverxref@cdfa6fd8dce638f3e1d1d80b07b4cf2b1530a7fe
```

```suggestion
sphinx-hoverxref==1.3.0
```
sphinx-hoverxref 1.3.0 was released with the necessary updates :tada:

---

https://github.com/DisnakeDev/disnake/pull/788/commits/ecb80a75bb3d614a61a1bb9951eaa2a0547dde23